<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vocabulary Master</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/res/mainwindow/mainwindow.png</normaloff>:/res/mainwindow/mainwindow.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>258</width>
      <height>213</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="answerLayout">
     <item>
      <widget class="QLabel" name="language2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="window2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>258</width>
      <height>213</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="questionLayout">
     <item>
      <widget class="QLabel" name="language1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="window1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="category">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>268</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="recordControls">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="priority1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority3">
       <property name="text">
        <string>3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority4">
       <property name="text">
        <string>4</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority5">
       <property name="text">
        <string>5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority8">
       <property name="text">
        <string>8</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="priority9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="recordEnabled">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOrganizer"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuVocabularies">
    <property name="title">
     <string>&amp;Vocabularies</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionLicense"/>
    <addaction name="actionSendFeedback"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuVocabularies"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionNext"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionAnswer"/>
   <addaction name="actionFindInVocabulary"/>
   <addaction name="separator"/>
   <addaction name="actionMute"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/menubar/exit.png</normaloff>:/res/mainwindow/menubar/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/menubar/settings.png</normaloff>:/res/mainwindow/menubar/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="actionOrganizer">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/menubar/new.png</normaloff>:/res/mainwindow/menubar/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Organizer...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/res/mainwindow/menubar/open.png</normaloff>:/res/mainwindow/menubar/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="qaManage">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/menubar/manage.png</normaloff>:/res/mainwindow/menubar/manage.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage...</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/toolbar/play.png</normaloff>:/res/mainwindow/toolbar/play.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/toolbar/stop.png</normaloff>:/res/mainwindow/toolbar/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/toolbar/next.png</normaloff>:/res/mainwindow/toolbar/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/toolbar/mute.png</normaloff>:/res/mainwindow/toolbar/mute.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionAnswer">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/toolbar/answer.png</normaloff>:/res/mainwindow/toolbar/answer.png</iconset>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/menubar/about.png</normaloff>:/res/mainwindow/menubar/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/menubar/license.png</normaloff>:/res/mainwindow/menubar/license.png</iconset>
   </property>
   <property name="text">
    <string>&amp;License...</string>
   </property>
  </action>
  <action name="actionFindInVocabulary">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/mainwindow/toolbar/findinvocabulary.png</normaloff>:/res/mainwindow/toolbar/findinvocabulary.png</iconset>
   </property>
   <property name="text">
    <string>Find in vocabulary</string>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="text">
    <string>Send &amp;feedback...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
